CHANGELOG.md

```diff
@@ -66,7 +66,7 @@ Released on June 28, 2022
 * QueryDataset class #53
 * Unit tests for NeuralNet class #86
 * Error message if you pick the wrong metrics #110
-* Unit tests for HDF5DataSet class parameters #82
+* Unit tests for GraphDataset class parameters #82
 * Installation from PyPI in the readme #122
 
```

Comment:
should this be updated, given that when the logged change happened it was still called HDF5DataSet?

Reply:
The CHANGELOG should reflect the code at the time of the writing, so we can leave HDF5DataSet. In the new entry of it we'll write that we changed the class' name.